tests/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor.Tests/MappedSubTextTestData.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Razor
```

Comment:
I'm assuming this is generated. Can you make it clear so that we don't change it manually? Maybe comments at the top or a directory with RazorGeneratedSource or something

Reply:
Yep, good point. I'll add a comment.